core/src/main/java/org/apache/calcite/adapter/enumerable/RexToLixTranslator.java

```diff
@@ -741,8 +757,8 @@ public RexNode deref(RexNode expr) {
   /** Translates a call to an operator or function. */
   private Expression translateCall(RexCall call, RexImpTable.NullAs nullAs) {
     final SqlOperator operator = call.getOperator();
-    CallImplementor implementor =
-        RexImpTable.INSTANCE.get(operator);
+    CallImplementor implementor = null;
```

Comment:
Could we have a comment explaining this change?

Reply:
Thanks for review!
This is the commit just for functionality.  As `translateRexCall` will be never called under new implementation, I just set null here to pass the compilation. For review convenience, I try to minimize the changes as possible. In latter commits (e.g., cosmetic and refactor), these unused parts will be removed.

Reply:
Ok, thanks for the clarification!